eq-author-api/package.json

```diff
@@ -43,7 +43,7 @@
     "pg-hstore": "latest",
     "pino-noir": "latest",
     "uuid": "latest",
-    "wait-for-postgres": "latest"
+    "wait-on": "^3.2.0"
```

Comment:
Can we point this on `latest`? 

It should just be a case of updating this and running `yarn`.